var/spack/repos/builtin/packages/rkt-racket-lib/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class RktRacketLib(RacketPackage):
+    """Stub package for packages which are currently part of core racket installation (but which may change in the future)."""
+
+    git      = "ssh://git@github.com/racket/racket.git"
+    
+
+    maintainers = ['elfprince13']
```

Comment:
Can you expand on this? What is the purpose of this stub package (not really familiar with racket).

Reply:
Basically the Racket dev team added a dummy package for stuff that's currently baked into the binary, but which they can't guarantee won't eventually be factored out, and if you do a minimal install it doesn't get automatically marked as installed by `raco`.